.github/workflows/ci.yml

```
@@ -0,0 +1,168 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, codex/implement-phase-1-of-issue-#4 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)